frontend/src/App.js

```diff
@@ -26,6 +27,26 @@ const useStyles = makeStyles((theme) => ({
  */
 export default function App() {
   const classes = useStyles();
+  const matchDataRefreshRate = 5000;
+  const [matchStatus, setMatchStatus] = React.useState('Unmatched');
```

Comment:
Would you mind explaining this line to me ?

Reply:
Yes, this line is using the state hook in React. Essentially, it is declaring a new state variable for the matchStatus and a setter function for it. It is also setting the default match status to 'Unmatched' since the user is unmatched when first loading the application.